tests/content-block-manager.spec.js

```diff
@@ -0,0 +1,133 @@
+import { expect } from "@playwright/test";
+import { test } from "../lib/cachebust-test";
+import { publishingAppUrl, waitForUrlToBeAvailable } from "../lib/utils";
+
+test.describe("Content Block Manager", { tag: ["@app-content-object-store"] }, () => {
+  let embedCode;
+  let value;
+
+  const contentBlockPath = `${publishingAppUrl("whitehall-admin")}/content-block-manager/content-block/18`;
+  const whitehallPath = "/government/admin/news/1658299";
```

Comment:
I think it would be really helpful to be able to run these locally - wonder if having the ID's as environment variables or variables on their own would help us swap them out for ones we have setup locally?

Reply:
I had a go at this, and it was a bit fiddly because we don't have signon locally, and we also need to ensure the state is correct locally too. Will merge this for now and then can look at improving later.